build.yaml

```diff
@@ -2341,6 +2341,9 @@ steps:
       - name: dedup-job-resources
         script: /io/sql/dedup_job_resources.py
         online: true
+      - name: fix-job-groups-state-enum
```

Comment:
Notice that this should be okay that I inserted it here as the subsequent populate job groups migration failed and is not in the database yet as a completed migration.